control/tests/statesp_test.py

```diff
@@ -341,18 +341,14 @@ def test_evalfr(self, dt, omega, resp):
         else:
             s = omega * 1j
 
-        # Correct version of the call
+        # Correct versions of the call
```

Comment:
#449 introduces a `test_freqresp_depracated` into statesp_test.py which is not introduced here.
#449 introduces a `test_call` into statesp_array_test.py. That file does not exist anymore, the test should be introduced into  statesp_test.py.

Reply:
Checked and slightly modified in 7acfc779f10806610db0680c3e305ab9e26beb08